chainstate/src/detail/tests/mod.rs

```diff
@@ -162,3 +164,27 @@ fn create_new_outputs(tx: &Transaction) -> Vec<(TxInput, TxOutput)> {
         .filter_map(move |(index, output)| create_utxo_data(&tx.get_id(), index, output))
         .collect::<Vec<(TxInput, TxOutput)>>()
 }
+
+// generate 5 regtest blocks and print them in hex
+#[ignore]
+#[test]
+fn generate_blocks_for_functional_tests() {
```

Comment:
Please add a comment that this should be removed when block production is complete and is usable through RPC

Reply:
Another TODO :-)

Reply:
For each item I remove the TODO list, I add 1-3 more. I need to start working *really* fast :smiley: